packages/embed/README.md

```diff
@@ -14,12 +14,3 @@ npm run start:stage
 # Run against production Audius services
 npm run start:prod
 ```
-
-## Deploying
```

Comment:
Minor nit: Is it worth mentioning here that deployment happens through CI? Or is that just generally known?

Reply:
i think just generally known, but i can add some notes